packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,122 @@
+      },
+    ],
+  },
+}
 ```
 
->**Note:** Review the complete use of `idx.recoverPassword` in the [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxrecoverpassword).
-
-### 2: Create the reset password authenticators page
-
-The next step is to create a page that shows the authenticator key returned from the `handleTransaction()` method. For this use case, the email authenticator is returned (`authenticator: AuthenticatorKey.OKTA_EMAIL`). The page includes the name of the returned authenticator and the ability to select the authenticator to initiate the authentication verification process.
+Using this response, display a list of authenticators to the user. If you've configured your org as described in [Configuration updates](#configuration-updates), email is included in the list of available authenticators.
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![Displays the Select authenticator page that has a field to select email.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-select-auth-nodejs.png)
+![Screenshot of a page displaying a list of authenticators including the email authenticator](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-select-auth-nodejs.png)
 
 </div>
 
-### 3: The user selects the email authenticator factor
-
-After the user selects the email authenticator, the next step is to call `idx.recoverPassword` with the authenticator type, as shown in the `authenticator.js` file of the SDK sample application. The method returns a status of `Idx.Status:PENDING`. This status indicates that the Okta platform has emailed the verification code to the user's email address and is now awaiting verification. The response also includes a `nextStep` field that requires an email verification code input parameter.
-
-```JavaScript
-router.post('/select-authenticator', async (req, res, next) => {
-  const { idxMethod } = req.getFlowStates();
-  const { authenticator } = req.body;
-  const authClient = getAuthClient(req);
-  const transaction = await authClient.idx[idxMethod]({ authenticator });
-  handleTransaction({ req, res, next, authClient, transaction });
-});
+### 5. Verify identity with the email authenticator
+
+Next, the user selects and verifies their identity with the email authenticator challenge. The email authenticator supports OTP and magic links, and you can integrate both methods into your application. Learn more about integrating the email authenticator challenge by visiting the [Okta email integration guide](/docs/guides/authenticators-okta-email/nodeexpress/main/#integrate-email-challenge-with-magic-links).
+
+### 6. Display password reset page
+
+After the user verifies their email, `OktaAuth.idx.proceed()` returns a response indicating that user is allowed to reset their password. Specifically, `IdxTransaction` returns a `status` of `PENDING` and `nextStep.name` equal to `reset-authenticator`.
```

Comment:
Just to keep this consistent with step 4. Again, don't know where IdxTransaction came from, so suggested an alternative for that, but if you explain it elsewhere, revert that if you like.

```suggestion
After the user verifies their identity, `OktaAuth.idx.proceed()` returns a response indicating that user is allowed to reset their password. Specifically, the response includes:

-  `status` = `PENDING`
-  `nextStep.name` = `reset-authenticator`

```

Reply:
Updated formatting, though lets talk about "=" vs "equal to"

Reply:
rewrote into a sentence.